proprietary/design-tokens/studio/common/viewport (beta)/lg.json

```diff
@@ -0,0 +1,136 @@
+{
+  "figma": {
```

Comment:
wordt nog opgepakt door de parser, resulterend in een `--figma`-variabele

Reply:
Klopt. Wordt opgelost in #92